RevenueCat.xcodeproj/project.pbxproj

```diff
@@ -3168,6 +3168,7 @@
 				SDKROOT = iphoneos;
 				SUPPORTS_MACCATALYST = YES;
 				SWIFT_OPTIMIZATION_LEVEL = "-Onone";
+				SWIFT_STRICT_CONCURRENCY = targeted;
```

Comment:
We'll see if we keep this or not.

Reply:
I think it's not a bad idea to keep this so we ensure we maintain this level of correctness prior to Swift 6.0